homeassistant/components/plugwise/climate.py

```diff
@@ -74,20 +68,17 @@ def __init__(
 
         self._loc_id = coordinator.data.devices[device_id]["location"]
 
-    @plugwise_exception_handler
+    @plugwise_command
     async def async_set_temperature(self, **kwargs: Any) -> None:
         """Set new target temperature."""
-        temperature = kwargs.get(ATTR_TEMPERATURE)
-        if (temperature is not None) and (
+        if ((temperature := kwargs.get(ATTR_TEMPERATURE)) is not None) and (
```

Comment:
I'd invert this check and raise if true. Then we can outdent below.

Reply:
Done